workflows/gitops-promotion/versions/0.0.1/README.md

```diff
@@ -0,0 +1,18 @@
+# gitops-promotion
+
+## Summary
+
+Templates for doing GitOps-style promotions of microservices from one K8s environment to another. Assumes that you have a GitOps monorepo containing subdirectories that represent environments, each of which contains the manifests for deploying that environment to a K8s cluster. Supported manifest types include Kustomize, Helm Charts, and raw YAML files.
+
+## Templates
+
+1. [promote-from-src-to-dest-env](./docs/promote-from-src-to-dest-env.md)
```

Comment:
please give the full path. it's not working well when accessing it from argo hub

Reply:
Will do!